### PR TITLE
Mail2kvanitha/c01-aws01

### PR DESCRIPTION
**Pull request by mail2kvanitha**
# Description

Exercise (e.g. C01-E01): c01-aws01

Student Name: Vanitha Kaliyaperumal

> A brief explanation of what has been done.
Create 2 instances and connect from one instance with Public IP to another instance without public IP.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

**Comment by mail2kvanitha** [comment]
projects/project02/application/web/public/favicon.ico file removed from projects

File: classes/01class/exercises/c01-aws01/mail2kvanitha/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## EC2 (c01-aws01)
+
+- Use the CLI to create a new EC2 Instance inside the default VPC with a public IP address and connect to it though SSH.
+- Create another EC2, also using the CLI instance in the same VPC but with private address only and connect to it.
+Steps to connect from one instance with Public IP to another instance with no Public IP.
+STEP1:CREATE SECURITY GROUP:
+----------------------------
+aws ec2 create-security-group --group-name webserver-sg --description "Webserver Security Group"
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/mail2kvanitha/README.md:8]: can you provide what rules have been created in this security group and which command was used to create those rules?

File: classes/01class/exercises/c01-aws01/mail2kvanitha/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+STEP4:CONNECT FROM INSTANCE1 to INSTANCE2:
+------------------------------------------
+Copy the WEBSERVER.pem file to INSTANCE1.
+scp -i WEBSERVER.pem WEBSERVER.pem ec2-user@<Public DNS name>:~
+
+SSH to INSTANCE1 using the keypair
+ssh -i WEBSERVER.pem ec2-user@<EC2 Instance1 Public DNS>
+
+SSH to INSTANCE2 using the keypair copied 
+ssh -i WEBSERVER.pem ec2-user@<EC2 Private IP>
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/mail2kvanitha/README.md:32]: from where are you accessing this instance?

File: classes/01class/exercises/c01-aws01/mail2kvanitha/README.md (hunk 1 above)
Comment (mail2kvanitha) [reply classes/01class/exercises/c01-aws01/mail2kvanitha/README.md:13]: Updated the README.md with required security groups and rules added.

File: classes/01class/exercises/c01-aws01/mail2kvanitha/README.md (hunk 2 above)
Comment (mail2kvanitha) [reply classes/01class/exercises/c01-aws01/mail2kvanitha/README.md]: Accessing instance from Terminal using SSH Client

**Comment by mail2kvanitha** [comment]
Hi Kiko, Denis, Caio,
Please see below screenshot of README.md file for c01-aws01 and c01-aws02.
Is there any different formatting expected than below. Else, I can only put
the aws commands in the file and delete other line. Please let me know.
*c01-aws01*
[image: image.png]


*c01-aws02*

[image: image.png]

**Pull request merged**